packages/utils/src/string.js

```diff
@@ -32,3 +33,10 @@ export const getArticle = word => {
   }
   return word.match(/^[aeiou]/i) ? 'an' : 'a';
 };
+
+const truncate = (string, length) => (string.length <= length ? string : string.slice(0, length));
+
+export const stringToFilename = string => {
+  const sanitized = sanitize(string).replace(/\s+/g, '-').toLowerCase().toLowerCase();
```

Comment:
Only 1 `toLowerCases()` should be enough?

```suggestion
  const sanitized = sanitize(string).replace(/\s+/g, '-').toLowerCase();
```